tests/system/Debug/Toolbar/Collectors/DatabaseTest.php

```diff
@@ -23,9 +23,7 @@ final class DatabaseTest extends CIUnitTestCase
     public function testDisplay(): void
     {
         /** @var MockObject&Query $query */
-        $query = $this->getMockBuilder(Query::class)
-            ->disableOriginalConstructor()
-            ->getMock();
+        $query = $this->createMock(Query::class);
```

Comment:
it seems cause error:

```bash


1) CodeIgniter\Debug\Toolbar\Collectors\DatabaseTest::testDisplay
Failed asserting that two strings are identical.
--- Expected
+++ Actual
@@ @@
-'ROOTPATH/tests/system/Debug/Toolbar/Collectors/DatabaseTest.php:35'
+'ROOTPATH/tests/system/Debug/Toolbar/Collectors/DatabaseTest.php:33'
```

Reply:
Fixed with update line as it read `__FILE__` https://github.com/codeigniter4/CodeIgniter4/pull/6033/commits/b2f228f00292226bcca32153fc4a96d32526dcf9